packages/config/config/devices/0x0234/matrix_zba7140.json

```diff
@@ -0,0 +1,634 @@
+// Logic Group MATRIX ZBA7140
+// Matrix ZBA7140
+{
+	"revision": 4,
+	"manufacturer": "Logic Group",
+	"manufacturerId": "0x0234",
+	"label": "MATRIX ZBA7140",
+	"description": "Matrix ZBA7140",
```

Comment:
Remove

Reply:
why should i remove these lines? how do you define the product then ?

Reply:
Oh this is so dumb... I only selected line 8 for my comment, but you can't see that I didn't reference the other lines too.
The description is an exact duplicate of the label, so it should be removed.